integration_tests/tests/s3.rs

```diff
@@ -527,13 +529,29 @@ fn test_get_object_with_presigned_url(region: &Region, credentials: &AwsCredenti
     assert!(buf.len() > 0);
 }
 
+fn test_get_object_with_expired_presigned_url(region: &Region, credentials: &AwsCredentials, bucket: &str, filename: &str) {
+    let req = GetObjectRequest {
+        bucket: bucket.to_owned(),
+        key: filename.to_owned(),
+        ..Default::default()
+    };
+    let opt = PreSignedRequestOption {
+        expires_in: ::std::time::Duration::from_secs(1)
+    };
+    let presigned_url = req.get_presigned_url(region, credentials, &opt);
+    ::std::thread::sleep(::std::time::Duration::from_secs(1));
```

Comment:
You probably want to wait for a bit longer than a seconds, just in case the client or server clock is a bit skewed.